src/gateway/environments/jsdom-sixteen/jsdom-sixteen-environment.js

```diff
@@ -52,7 +53,10 @@ export class JSDOMSixteenEnvironment implements IRenderEnvironment {
      */
     constructor(configuration: IJSDOMSixteenConfiguration) {
         if (configuration == null) {
-            throw new Error("Must provide environment configuration");
+            throw new KAError(
+                "Must provide environment configuration",
+                Errors.Internal,
```

Comment:
InvalidInput here too.

Reply:
Again, this is not due to invalid input to the service, only in coding, so I don't believe it fits InvalidInput